docs/meshstack.index.md

```diff
@@ -13,6 +13,12 @@ It is NOT a meta-layer across clouds. Users access native cloud APIs for their d
 
 ![meshStack Context and Components](assets/meshstack-chart.svg)
 
+##Central Concepts
+
+meshStack uses the concepts of Workspaces and Projects to represent the structure of existing application teams in the meshStack platform. Application Team can manage Tenants inside Projects. In order to reflect the organizations compliance and securiity standards in the cloud a Landing Zone is applied to every Tenant. Building Blocks can be leverage either as part of a mudular Landing Zone or additionally on top of a Tenant. Building Blocks are deployed based on the blueprint provided via the Building Block definition. 
+meshStack supports a multi platform operation which means you can manage multiple platforms of the same or different typ with meshStack. Below you can see an overview of the relations between the individual elements of meshStack as objects.
+![meshModel](assets/meshModel.png)
```

Comment:
So the problem seems you added a reference to the image but you didn't upload the image. Check your VSCode, it might be hanging around somewhere there.